_i18n/en/_docs/javascript-api.md

```diff
@@ -846,7 +847,9 @@ Interceptor.attach(f, {
     `glob` and returns their addresses as an array of [`NativePointer`](#nativepointer)
     objects.
 
-+   `DebugSymbol.load(path)`: loads debug symbols for a specific module.
++   `DebugSymbol.load(path)`: loads debug symbols for a specific module. This feature is
+    not fully supported in Windows modules, {modulename}.pdb file must downloaded and 
+    located in the project folder.
```

Comment:
> This feature is not fully supported in Windows modules

That's confusing, since Windows is the only backend where we implement this API.

> file must downloaded and located in the project folder.

This sounds incorrect. Is the intention to say that it must be located in the same directory as the .dll?